apps/commcare/models.py

```diff
@@ -24,6 +24,12 @@ class CommCareServer(BaseModel):
     def __str__(self):
         return f'{self.name} ({self.url})'
 
+    def get_url_base(self):
```

Comment:
I have often found a function like this one to be useful:
```python
def slash_join(*args: str) -> str:
    """
    Joins arguments with a single ``/``. Useful for concatenating
    strings to form a URL.

    >>> slash_join('https://example.com/', '/foo')
    'https://example.com/foo'
    >>> slash_join('https://example.com', 'api/', '/resource-type/')
    'https://example.com/api/resource-type/'

    """
    if not args:
        return ''
    append_slash = args[-1].endswith('/')
    joined = '/'.join([arg.strip('/') for arg in args])
    return joined + '/' if append_slash else joined
```

Reply:
ah, that does look handy!